.github/workflows/release.yml

```diff
@@ -1,111 +1,56 @@
-name: release Boltzmann
+name: Release Boltzmann
```

Comment:
This workflow is untested for the time being. I think it'll need to land on the primary branch to run, but I wanted to give you a chance to review before landing it.